test/errors/nonIterable-detailed.good

```diff
@@ -0,0 +1,31 @@
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelRange.chpl:2277 ───
+  Where clause does not result in a param bool value
+
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelRange.chpl:2294 ───
+  Where clause does not result in a param bool value
+
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelRange.chpl:2333 ───
+  Where clause does not result in a param bool value
+
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelRange.chpl:2362 ───
+  Where clause does not result in a param bool value
+
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelRange.chpl:2391 ───
+  Where clause does not result in a param bool value
+
+─── error in nonIterable.chpl:2 [NonIterable] ───
+  Cannot iterate over value of type 'int(64)'.
+  In the following loop:
+      |
+    2 | for i in x {}
+      |          ⎺    
+      |
+
+─── error in nonIterable.chpl:3 [NonIterable] ───
+  Cannot iterate over param of type 'int(64)'.
```

Comment:
This is another case where value vs. param is irrelevant and feels like it may lead a reader down the wrong mental path ("Oh, if I change it to a non-param I'll be OK?")